docs/features/datastores/overview.md

```diff
@@ -0,0 +1,31 @@
+# Demystifying Datastorses
+
+## What are Datastores
+
+Datastores are high-performance, low-latency, versioned, and scalable datasets.
+
+## What files can be accepted
+Datastores can be created from any file type
+
+## Why use Datastores
+Datastores are zero cost datasets backed by cloud storage. **Add additional content as to why this is better than local directory storage etc**.
```

Comment:
Make training iteration quicker, because no need to download the data for your script every time. And data is local to run/session machine

Reply:
not exactly true you still have to transfer the data from S3 to the Session or Run which makes Datastores worse than having the data on local disk. We need to be careful here.

Reply:
The only obvious benefit is when the data is too large for your local machine. But even then you can lazily load it sometimes so we should be very intentional and thorough here. @rlizzo input will be important for the community.

Reply:
we're saying the same thing, i think.. at least about the datastore access - no need to download it from your script on every iteration. but not sure i fully agree with the caching - do we have that yet? @rlizzo